src/core/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import { Tooltip as MuiTooltip } from '@material-ui/core'
+import React, { ReactElement } from 'react'
+import { IDefault } from './Advertise'
+
+interface IProps extends IDefault {
+    placement?: 'bottom-end',
+                'bottom-start',
+                'bottom',
+                'left-end',
+                'left-start',
+                'left',
+                'right-end',
+                'right-start',
+                'right',
+                'top-end',
+                'top-start',
+                'top',
```

Comment:
Is it valid in TS?

I would rather do:
```typescript
placement?: 
    | 'bottom-end'
    | 'bottom-start'
    | 'bottom'
    | 'left-end'
    | 'left-start'
    | 'left'
    | 'right-end'
    | 'right-start'
    | 'right'
    | 'top-end'
    | 'top-start'
    | 'top',
```

Reply:
Solved